### PR TITLE
GNIRS static configuration

### PR DESCRIPTION
**Pull request by jdnavarro**
I could only identify *well depth* as static. The rest of the parameters seem to be dynamic.

I already started working with the dynamic configuration but since it seems it's going to take me a while I'm sending the static changes first.

### CODE REVIEW

File: modules/db/src/main/scala/gem/dao/StaticConfigDao.scala [hunk 1]
```diff
@@ -116,21 +116,19 @@ object StaticConfigDao {
...
     object F2 {
 
-      import Instrument.Flamingos2
-
       def select(sid: Int): Query0[StaticConfig.F2] =
         sql"""
           SELECT mos_preimaging
             FROM static_f2
-           WHERE static_id = $sid AND instrument = ${Flamingos2: Instrument}
+           WHERE static_id = $sid
```
Comment (cquiroz) [inline modules/db/src/main/scala/gem/dao/StaticConfigDao.scala]: Why this change?

Comment (jdnavarro) [reply modules/db/src/main/scala/gem/dao/StaticConfigDao.scala]: While writing the same code for GNIRS we realized there is already postgres constraint for the instrument (`is_f2`), so the extra `AND` is not needed.

File: modules/core/shared/src/test/scala/gem/config/Arbitraries.scala [hunk 2]
```diff
@@ -122,6 +121,15 @@ trait Arbitraries {
       } yield StaticConfig.GmosSouth(c, s)
     )
 
+  implicit val arbGnirsWellDepth: Arbitrary[GnirsWellDepth] =
+    Arbitrary(
+      Gen.oneOf(GnirsWellDepth.Shallow,
+                GnirsWellDepth.Deep)
+    )
```
Comment (swalker2m) [inline modules/core/shared/src/test/scala/gem/config/Arbitraries.scala]: There is an `Enumerated` instance for `GnirsWellDepth` so I don't think this needs to be explicitly defined if `ArbEnumerated.arbEnumerated` is in scope.
